app/gateway/changelog.md

```diff
@@ -2206,6 +2206,21 @@ openid-connect
 * Bumped `lodash` for Dev Portal from 4.17.11 to 4.17.21
 * Bumped `lodash` for Kong Manager from 4.17.15 to 4.17.21
 
+## 2.8.4.2
```

Comment:
@acgoldsmith

Shall we add the two lines at https://github.com/Kong/kong-ee/blob/next/2.8.x.x/CHANGELOG-EE.md?plain=1#L11 to this PR?